Layout w HTML i CSS/product_page/solution.txt

```diff
@@ -0,0 +1,8 @@
+1. Podział stronu na 3 sekcje (header, gallery, description).
+2. Dodanie elementu nav i ul w headerze.
+3. Dodanie favicona i zdjecia w headerze.
+4. Utworzenie sekcji gallery.
+5. Utworzenie sekcji description.
+6. Dodanie nagłówków i paragrafu w sekcji description.
+7. Ustawienie elementów poszczególnych komponentów.
+8. Ostylowanie headera, sekcji gallery i description.
```

Comment:
Przy rozkładaniu widoku na czynniki pierwsze dobrze byłoby myśleć od razu o zadaniach, które się za tym kryją, niekoniecznie ze szczegółami implementacyjnymi.

Przykład
---
Feature title: "Implement Navbar desktop"

Tasks:
1. Write HTML skeleton (+ add assets, eg. page logo, icons itp).
2. Add positioning styles (navbar inner layout and component position on the screen)
3. Style links, logo, icons
---

Ogólnie bardzo fajnie :v: